recipes/cython-cmake/recipe.yaml

```diff
@@ -0,0 +1,58 @@
+context:
+  name: cython-cmake
+  version: 0.2.0
+  username: scikit-build
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com//${{ username }}/${{ name|lower }}/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 0673545871646e1736a2aaaf29f0f96ad1033da8fa6e457ee1cb99289048b8ac
+
+build:
+  script: ${{ PYTHON }} -m pip install --no-build-isolation --no-deps -vv .
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.9
```

Comment:
Don't we depend on 3.8+ currently? I don't think I've dropped 3.8 yet. Not that it matters, since 3.8 builds aren't being made on conda-forge anymore, but it seems like it would be better to match so greyskull isn't confused.

Reply:
Thank you for your advice.
I cared about specific deals with `importlib_resources` when `python<3.9`.
However, such a configuration seems to be applied even in the `recipe.yaml`, so I will try to modify it.

Reply:
You can't have conditional deps in noarch conda packages. Conda doesn't treat Python specially. So just requiring conditional imports all the time is correct.